pkg/trimaran/resourcestats.go

```diff
@@ -195,24 +195,24 @@ func GetNodeRequestsAndLimits(podInfosOnNode []*framework.PodInfo, node *v1.Node
 			// get requests and limits for pod
 			requested = GetResourceRequested(p)
 			limits = GetResourceLimits(p)
-			// make sure limits not less than requests
-			SetMaxLimits(requested, limits)
 		}
-		reqCpu := requested.MilliCPU
-		reqMem := requested.Memory
-		limitCpu := limits.MilliCPU
-		limitMem := limits.Memory
+		// make sure limits not less than requests
+		SetMaxLimits(requested, limits)
+
 		// accumulate
-		nodeRequest.MilliCPU += reqCpu
-		nodeRequest.Memory += reqMem
-		nodeLimit.MilliCPU += limitCpu
-		nodeLimit.Memory += limitMem
+		nodeRequest.MilliCPU += requested.MilliCPU
+		nodeRequest.Memory += requested.Memory
+		nodeLimit.MilliCPU += limits.MilliCPU
+		nodeLimit.Memory += limits.Memory
 	}
 	// cap requests by node capacity
 	setMin(&nodeRequest.MilliCPU, capCpu)
 	setMin(&nodeRequest.Memory, capMem)
 	setMin(&nodeRequestMinusPod.MilliCPU, capCpu)
 	setMin(&nodeRequestMinusPod.Memory, capMem)
+	// cap limits by node capacity
+	setMin(&nodeLimit.MilliCPU, capCpu)
```

Comment:
No need to cap the values of limits to node capacities. It is valid for a user to specify a limit that is higher than node capacity. This case is handled in [computeRisk()](https://github.com/kubernetes-sigs/scheduler-plugins/blob/6df71a420673837e2aaf90540f67d3cac18a799a/pkg/trimaran/lowriskovercommitment/lowriskovercommitment.go#L206).

Reply:
Sure @atantawi , reversed this change.